src/scss/prefix-suffix.scss

```diff
@@ -1,103 +1,116 @@
-@if not $o-forms-is-silent {
+@mixin oFormsAffixWrapper($class: 'o-forms') {
+	.#{$class}__affix-wrapper {
+		display: table;
+		width: 100%;
+		box-sizing: border-box;
+	}
 
-.o-forms-affix-wrapper {
-	display: table;
-	width: 100%;
-	box-sizing: border-box;
-}
+	.#{$class}__affix-wrapper .#{$class}__text,
+	.#{$class}__affix-wrapper .#{$class}__select {
+		display: table-cell;
+		border-radius: 0;
+	}
 
-.o-forms-affix-wrapper .o-forms-text,
-.o-forms-affix-wrapper .o-forms-select {
-	display: table-cell;
-	border-radius: 0;
-}
+	.#{$class}__affix-wrapper .#{$class}__text:first-child,
+	.#{$class}__affix-wrapper .#{$class}__select:first-child {
+		border-top-left-radius: $_o-forms-field-border-radius;
+		border-bottom-left-radius: $_o-forms-field-border-radius;
+	}
 
-.o-forms-affix-wrapper .o-forms-text:first-child,
-.o-forms-affix-wrapper .o-forms-select:first-child {
-	border-top-left-radius: $_o-forms-field-borderradius;
-	border-bottom-left-radius: $_o-forms-field-borderradius;
+	.#{$class}__affix-wrapper .#{$class}__text:last-child,
+	.#{$class}__affix-wrapper .#{$class}__select:last-child {
+		border-top-right-radius: $_o-forms-field-border-radius;
+		border-bottom-right-radius: $_o-forms-field-border-radius;
+	}
 }
 
-.o-forms-affix-wrapper .o-forms-text:last-child,
-.o-forms-affix-wrapper .o-forms-select:last-child {
-	border-top-right-radius: $_o-forms-field-borderradius;
-	border-bottom-right-radius: $_o-forms-field-borderradius;
-}
+@mixin oFormsAffixButton($class: 'o-forms') {
+	.#{$class}__prefix button,
+	.#{$class}__suffix button {
+		border-width: 0;
+		margin: $_o-forms-field-default-padding-top * -1 $_o-forms-field-default-padding-leftright * -1 $_o-forms-field-default-padding-bottom * -1;
+		padding: $_o-forms-field-default-padding-top $_o-forms-field-default-padding-leftright $_o-forms-field-default-padding-bottom;
+		background-color: transparent;
+		cursor: pointer;
+		font: inherit;
+		color: inherit;
+		// scss-lint:disable VendorPrefixes VendorPrefix
```

Comment:
Nope, it only disables the linting for the remainder of the block: https://github.com/brigade/scss-lint#disabling-linters-via-source